Work

tingamapuro04 opened the pull request:
I am working on the work branch where i have introduced three more containers to the mobile skeleton. The first container describe my work and skills. In this container i have added four sub-sections with each sub-section having an image at the top and then followed by the body and then a button. I have used grid to place these four sub-sections.
In the second container I have introduced contains information about self generally . In the last container contains contact information and i have also included a form for user input.
I have continued to work the files such as index.html and nina.css

tingamapuro04 committed:
Added a second container

tingamapuro04 committed:
Added four countainers inside

tingamapuro04 committed:
Added images files

tingamapuro04 committed:
filled the first card

tingamapuro04 committed:
Added second container of s.container

tingamapuro04 committed:
Added two containers to s.container

tingamapuro04 committed:
Added About container

tingamapuro04 committed:
Added the last contact page

tingamapuro04 committed:
linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 commented:
I have generally acted upon linter errors.

koomekevo reviewed (approved):
Hi @tingamapuro04 , 

<img src="https://media0.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly implement the works section as suggested in the below screenshot

![tingacorrect](https://user-images.githubusercontent.com/87413774/166084335-28d2d412-523b-42cc-b190-9c2ab238ff48.png)

The below screenshot is your design

![tingaworksectionwrong](https://user-images.githubusercontent.com/87413774/166084358-2619131d-fb95-4109-ad5e-88925af620c2.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

koomekevo reviewed (changes requested):
Hi @tingamapuro04 , 

<img src="https://media0.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly implement the works section as suggested in the below screenshot

![tingacorrect](https://user-images.githubusercontent.com/87413774/166084335-28d2d412-523b-42cc-b190-9c2ab238ff48.png)

The below screenshot is your design

![tingaworksectionwrong](https://user-images.githubusercontent.com/87413774/166084358-2619131d-fb95-4109-ad5e-88925af620c2.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Create CNAME

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Merge branch 'work' of https://github.com/tingamapuro04/Portfolio into work

tingamapuro04 committed:
LInter corrections

tingamapuro04 committed:
Linter changes

tingamapuro04 committed:
LInters

tingamapuro04 committed:
Linters corrections

tingamapuro04 commented:
I am working on the work branch where i have introduced three more containers to the mobile skeleton. The first container describe my work and skills. In this container i have added four sub-sections with each sub-section having an image at the top and then followed by the body and then a button. I have used grid to place these four sub-sections.
In the second container I have introduced contains information about self generally . In the last container contains contact information and i have also included a form for user input.
I have continued to work the files such as index.html and nina.css

I have added a wrap to the first and third cards in the class s-container.

vic778 commented on index.html:
```diff
@@ -35,14 +35,236 @@ <h1> <strong>I’m Mariane
+
+        <div class="box box-3">
+            <div class="snap3"></div>
+            <section class="body">
+                <header class="Tonic">
+                    <div class="title1">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="quali">
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="back">2015</li>
+                        </ul>
+                    </div>
+                </header>
+                <div class="para1">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="Tech">
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="java">javaScript</li>
+                    </ul>
+                </div>
+                <div class="Submission">
+                    <button type="submit">See Project</button>
+                </div>
+            </section>
```
`Figma template`
![Screenshot from 2022-04-30 09-42-37](https://user-images.githubusercontent.com/77842237/166095018-3d827650-a75e-44ed-b809-ccc4b8f19c15.png)
`your work`

![Screenshot from 2022-04-30 09-42-30](https://user-images.githubusercontent.com/77842237/166095231-27f08009-8d31-402b-b43d-cbd52450c54a.png)

- [ ] Kindly You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

vic778 reviewed (changes requested):
## Hello @tingamapuro04 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://user-images.githubusercontent.com/65068771/141811112-252360b9-757f-4279-bb3b-b7a2a4675ad3.gif)
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tingamapuro04 committed:
Fontise change on cards

tingamapuro04 committed:
deletion

tingamapuro04 committed:
Linter corrections

tingamapuro04 commented:
I am working on the work branch where i have introduced three more containers to the mobile skeleton. The first container describe my work and skills. In this container i have added four sub-sections with each sub-section having an image at the top and then followed by the body and then a button. I have used grid to place these four sub-sections.
In the second container I have introduced contains information about self generally . In the last container contains contact information and i have also included a form for user input.
I have continued to work the files such as index.html and nina.css

I have added a wrap to the first and third cards in the class s-container.

I have changed the font-sizes in the the class s-container.

KDlamini reviewed (approved):
### Hi @tingamapuro04 🙋‍♀️,  

Great work! 💯 
You have done a great job implementing the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

![giphyapproved](https://user-images.githubusercontent.com/38088818/166100633-bc087685-f7f7-4a97-bc86-e8f00e18c110.gif)
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
